bl/evaluation/printer.go

```diff
@@ -27,7 +25,7 @@ type Printer interface {
 
 type PrintResultsData struct {
 	Results               FormattedResults
-	RulesData             []cliClient.RuleData
+	AdditionalJUnitData   AdditionalJUnitData
```

Comment:
I think renaming this piece of code to `JUnit` data is a bit counter productive. This file is a printer that is responsible for printing all kinds of outputs and I can easily see this becoming an object containing output specific data for other outputs going forward.

Can we somehow pass this data directly to the `FormattedOutputToJUnitOutput`?